clarity/_version.py

```diff
@@ -361,7 +361,7 @@ def git_pieces_from_vcs(tag_prefix, root, verbose, runner=run_command):
             if verbose:
                 fmt = "tag '%s' doesn't start with prefix '%s'"
                 print(fmt % (full_tag, tag_prefix))
-            pieces["error"] = "tag '%s' doesn't start with prefix '%s'" % (
+            pieces["error"] = "tag '{}' doesn't start with prefix '{}'".format(
```

Comment:
The pyupgrade will upgrade to the .format style but not to fstring.  I guess it considers that to be the least opinionated change.

Our pylint precommit will actually enforce the use of fstring though... BUT... it's specifically disabled in the .pre-commit-config.yaml for this one file. I did this because it looked like _version.py is being generated by versioneer and it would need a lot of fstring changes.  I thought it best to let it stay as is. Hope that clarifies things :-)

Reply:
Ahha, I hadn't clocked what file this was. That all makes sense.

Versioneer is a bit of a pain, I'm in favour of switching to `setuptools_scm` as its much cleaner. I mentioned in another comment that #186 would help but its still not perfect as the resulting `_version.py` isn't compliant with Black.